mobile/salto/src/stores.ts

```diff
@@ -1,8 +1,11 @@
-import { readable, type Readable } from 'svelte/store';
+import { readable, writable, type Readable } from 'svelte/store';
 import { platform } from "@tauri-apps/plugin-os";
 
 const currentPlatform = platform();
 
 export const isMobile = readable(false, (set) => {
   set(currentPlatform == "ios" || currentPlatform == "android");
 });
+
+export const selectedCurrencyStored = writable<string>('usd');
+export const fiatCurrenciesStored = writable<string[]>([]);
```

Comment:
I believe only the `selectedCurrencyStored` should be a store. It is the value that will be used accross different pages of the app.
`fiatCurrenciesStored` is only used when selecting `selectedCurrencyStored` in the setting page. 
Just load it from the backend when you need it.